test/functional/lua/secure_spec.lua

```diff
@@ -47,7 +47,8 @@ describe('vim.secure', function()
       })
 
       local cwd = fn.getcwd()
-      if #cwd + 23 > 500 then
+      local msg = cwd .. pathsep .. 'Xfile is not trusted.'
+      if #msg >= 500 then
```

Comment:
Can we extract this into a shared `screen_width` variable, or perhaps `screen.width` (if that exists)?

Reply:
I think there is `screen._width`